src/stacks-hierarchy/Root_PurchaseOverviewScreen/Root_PurchaseOverviewScreen.tsx

```diff
@@ -83,7 +83,7 @@ export const Root_PurchaseOverviewScreen: React.FC<Props> = ({
     }
   }, [params?.refreshOffer]);
 
-  const closeModal = () => navigation.goBack();
+  const closeModal = () => navigation.pop();
```

Comment:
This fixes the bug @jorelosorio, this does however create the need to test the overview screen as well.🤘🏻

Thanks to @ingvisim🫡

Reply:
@adriawh it didn't work for me, however I tried `navigation.popToTop()` instead and worked properly for the case we have been discussing, not sure about other places, could you please check trying that?

Reply:
Hmm, I did reset the cache and have tried several times and just the regular pop works. I dont feel like the popToTop is the correct function for this use-case

Reply:
OK, let's wait then until QA test this, but on my case is not working, however popToTop worked.